README.md

```diff
@@ -8,6 +8,8 @@
 [![Python lint&test](https://github.com/sadikkuzu/squiral/actions/workflows/python-package.yml/badge.svg)](https://github.com/sadikkuzu/squiral/actions/workflows/python-package.yml)
 [![Publish Python Package](https://github.com/sadikkuzu/squiral/actions/workflows/python-publish.yml/badge.svg)](https://github.com/sadikkuzu/squiral/actions/workflows/python-publish.yml)
 
```

Comment:
Could you add `<br/>` into this 10th line?

Reply:
done

Reply:
Cool, thanks 👍🏼